chain/test_chain_deployments_tracer.go

```diff
@@ -14,19 +19,15 @@ type testChainDeploymentsTracer struct {
 	// results describes the results being currently captured.
 	results []types.DeployedContractBytecodeChange
 
-	// callDepth refers to the current EVM depth during tracing.
-	callDepth uint64
-
-	// evm refers to the EVM instance last captured.
-	evm *vm.EVM
+	// evm refers to the last tracing.VMContext captured.
+	evmContext *tracing.VMContext
 
 	// pendingCallFrames represents per-call-frame data deployment information being captured by the tracer.
 	// This is committed as each call frame succeeds, so that contract deployments which later encountered an error
 	// and reverted are not considered. The index of each element in the array represents its call frame depth.
 	pendingCallFrames []*testChainDeploymentsTracerCallFrame
 
-	// selfDestructDestroysCode indicates whether the SELFDESTRUCT opcode is configured to remove contract code.
-	selfDestructDestroysCode bool
+	nativeTracer *TestChainTracer
```

Comment:
just make sure to add a comment about what the native tracer is in all the tracers.